README.md

```diff
@@ -140,6 +140,19 @@ $ clang \
 -o example example.c
 ```
 
+The builds of the toolchain come packaged with two config files, Omax.cfg and OmaxLTO.cfg.
+When used, these config files enable several build optimisation flags. OmaxLTO.cfg enables LTO
+specific flags. These configs can be optionally passed using the `--config` flag. For example:
+
+```
+$ clang \
+example.c \
+...
+--config=Omax.cfg \
+--config=OmaxLTO.cfg \
+-o example
+```
+
```

Comment:
Could we also add a hint like it is possible to use these config files as an inspiration to create config file for your project by picking options from there and tuning the parameters as fits the project?

Reply:
Done!